runner/src/scenario/ceramic/model_instance.rs

```diff
@@ -293,6 +330,7 @@ impl CeramicModelInstanceTestUser {
         redis_cli: &redis::Client,
         should_create: bool,
         redis_postfix: Option<String>,
+        should_subscribe: bool,
```

Comment:
This might not be necessary - see my comment above. One user on each worker is already making sure that the corresponding node is subscribed to the models.